R/describe_missing.R

```diff
@@ -0,0 +1,112 @@
+#' @title Describe Missing Values in Data According to Guidelines
+#'
+#' @description Provides a detailed description of missing values in a data frame.
+#' This function reports both absolute and percentage missing values of specified
+#' variables.
+#'
+#' @inheritParams extract_column_names
+#' @param by Optional character string, indicating the names of one or more
+#' variables in the data frame. If supplied, the data will be split by these
+#' variables and summary statistics will be computed for each group. Useful
+#' for survey data by first reshaping the data to the long format.
+#' @param sort Logical. Whether to sort the result from highest to lowest
+#' percentage of missing data.
+#' @return A dataframe with the following columns:
+#'  - `variable`: Variables selected.
+#'  - `n_missing`: Number of missing values.
+#'  - `missing_percent`: Percentage of missing values.
+#'  - `complete_percent`: Percentage of non-missing values.
+#' @param ... Arguments passed down to other functions. Currently not used.
```

Comment:
Do we actually need `...`?

If we keep it, it should be positioned before `@return`

Reply:
We don't need it, but I had added it because I pass it to `data_select()` for full compatibility, which has it as argument. That said, current documentation for `?data_select()` is 
> Arguments passed down to other functions. Mostly not used yet.

So I can either keep it or remove it, as you wish.